pkgs/top-level/all-packages.nix

```diff
@@ -8372,6 +8372,16 @@ in
     binutils-arm-embedded = pkgsCross.arm-embedded.buildPackages.binutils;
   };
 
+  msp430GccSupport = callPackage ../development/misc/msp430/gcc-support.nix { };
+
+  msp430Newlib      = callPackage ../development/misc/msp430/newlib.nix { };
+  msp430NewlibCross = callPackage ../development/misc/msp430/newlib.nix {
+    inherit (pkgs.buildPackages.xorg) lndir;
+    newlib = pkgs.newlibCross;
```

Comment:
We really shouldn't use `pkgs.*Cross` in nixpkgs...almost ever. (@matthewbauer I told you this would happen!)

See what the others do with the special stdenv instead.

Reply:
> We really shouldn't use pkgs.*Cross in nixpkgs...almost ever.

Hm, it's only being used here because this package is wrapping/replacing it and being used in a place where `newlibCross` [is already being used](https://github.com/NixOS/nixpkgs/pull/58330/files#diff-036410e9211b4336186fc613f7200b12R10157), should that be done some other way?

Reply:
Hey I'm really sorry I straight up misread this as `pkgsCross.msp430...`. Everything you did is fine, though `pkgs.` is fine to skip too here.